src/components/qualityOptions.js

```diff
@@ -52,19 +52,16 @@ export function getVideoQualityOptions(options) {
     qualityOptions.push({ name: '360p - 420 kbps', maxHeight: 360, bitrate: 420000 });
 
     if (maxStreamingBitrate) {
-        let selectedIndex = -1;
+        let selectedIndex = 0;
```

Comment:
Since you removed the fallback below, we should probably init as `qualityOptions.length - 1`. Therefore, if there is no options less than `maxStreamingBitrate`, `selectedIndex` will stay unchanged.

Reply:
Done.